roqoqo/build.rs

```diff
@@ -313,6 +313,9 @@ impl<'ast> Visit<'ast> for Visitor {
                 if trait_name.as_str() == "ImplementedIn1point6" {
                     self.roqoqo_version_register.insert(id.clone(), 6);
                 }
+                if trait_name.as_str() == "ImplementedIn1point7" {
```

Comment:
Is this all that needs adding for a new `ImplementedIn...` trait? I though there was more

Reply:
I did too, but looking at the others that's all that is needed apparently.